src/.stories/basic.story.js

```diff
@@ -34,43 +24,40 @@ export default {
 const Template = (args) => <InfiniteCalendar {...args} />;
 
 export const DefaultConfiguration = Template.bind({});
-DefaultConfiguration.args = {
-  
-};
+DefaultConfiguration.args = {};
 
 export const InitiallySelectedDate = Template.bind({});
 InitiallySelectedDate.args = {
-  selected: addDays(today, 5)
-}
+  selected: addDays(today, 5),
+};
```

Comment:
I can see we have some format changes but I did not find any related dependency upgrade, is it caused by your editor?